power_grid_model_c/power_grid_model/include/power_grid_model/auxiliary/serialization/deserializer.hpp

```diff
@@ -248,7 +248,7 @@ struct BoolVisitor : DefaultErrorVisitor<BoolVisitor> {
 template <class T> struct ValueVisitor;
 
 template <std::integral T> struct ValueVisitor<T> : DefaultErrorVisitor<ValueVisitor<T>> {
-    static constexpr std::string_view static_err_msg = "Expect an interger.";
+    static constexpr std::string_view static_err_msg = "Expected an integer.";
```

Comment:
This contained a typo `interger`, fixed to `integer`. Irena encountered that

Reply:
This change I agree. But others not. 

Change from "expect" to "expected" is a bit confusing. I propose either keep the original, or changed them to "integer expected", as @Jerry-Jinfeng-Guo suggests.